test/models/organisation_test.rb

```diff
@@ -0,0 +1,15 @@
+require 'test_helper'
+
+describe Organisation do
+  it 'returns the state of an organisation' do
+    organisation = Organisation.new(
```

Comment:
Not all of these attrs are required for the test

```ruby
assert Organisation.new(organisation_state: 'live').live?, "Expected org to be live"
```

Reply:
Good point, now amended: https://github.com/alphagov/collections/pull/586/commits/e3a7756564a5bdabd0def6bb1669dab3329664ac#diff-207daaac6cfd16d598218e9fcc875105R6